tools/base/envoy_python.bzl

```diff
@@ -67,4 +69,33 @@ def envoy_py_binary(
         visibility = visibility,
     )
 
-    envoy_py_test(name, package, visibility, envoy_prefix = envoy_prefix)
+    if test:
+        envoy_py_test(name, package, visibility, envoy_prefix = envoy_prefix)
+
+def envoy_py_command(
```

Comment:
@htuch im kinda surprised there isnt anything in `rules_ptyhon` like this to automatically make a command from a python entrypoint/function etc

either way, i think this is pretty useful and removes a load of boilerplate